docs/en/reference/caching.rst

```diff
@@ -313,10 +61,12 @@ result cache driver.
 .. code-block:: php
 
     <?php
-    $cacheDriver = new \Doctrine\Common\Cache\PhpFileCache(
+    $cache = new \Symfony\Component\Cache\Adapter\PhpFilesAdapter(
+        'doctrine_results',
+        0,
         '/path/to/writable/directory'
     );
-    $query->setResultCacheDriver($cacheDriver);
+    $query->setResultCache($cache);
```

Comment:
Before it was `setResultCacheImpl()` and `setResultCacheDriver()`. Now both are `setResultCache` and duplicates.

Reply:
The first example sets the result cache on the global configuration while the second example sets it on the query object.

Yes, the method names have been streamlined, but the two code snippets still show different approaches.

Reply:
I missed that $config line. My bad.